bwscanner/attacher.py

```diff
@@ -135,7 +135,7 @@ def got_newconsensus(event):
 
 
 @defer.inlineCallbacks
-def connect_to_tor(launch_tor, circuit_build_timeout, control_port=None,
+def connect_to_tor(launch_tor, circuit_build_timeout, tordir=None, control_port=None,
```

Comment:
Minor nit: we're mostly using underscore separated variable names. It would be more consistent to use `tor_dir` instead of `tordir` as the variable name?